From 18fbf794ffc2cab9f4b8df3c4132b094fc4ef281 Mon Sep 17 00:00:00 2001
From: nicolas.dufresne <nicolas.dufresne@c587cffe-e639-0410-9787-d7902ae8ed56>
Date: Tue, 12 Oct 2010 17:39:18 +0000
Subject: [PATCH 1/2] Disable transient linking

As explained at
http://www.cmake.org/Wiki/CMake_FAQ#Why_do_I_have_unwanted_semicolons_.3B_in_my_compiler_flags.3F
if CMake links A to B and B to C, all the flags from A will be appended when
C is linked. This behaviour is wrong, but CMake provides a workaround which
is implemented in this patch.

git-svn-id: https://libproxy.googlecode.com/svn/trunk@774 c587cffe-e639-0410-9787-d7902ae8ed56
---
 libproxy/cmake/libproxy.cmk |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/libproxy/cmake/libproxy.cmk b/libproxy/cmake/libproxy.cmk
index 5625d4e..dc9fa0f 100644
--- a/libproxy/cmake/libproxy.cmk
+++ b/libproxy/cmake/libproxy.cmk
@@ -18,5 +18,6 @@ endif()
 set_property(SOURCE ${CMAKE_CURRENT_SOURCE_DIR}/proxy.cpp
              PROPERTY COMPILE_DEFINITIONS MODULEDIR="${MODULEDIR}";BUILTIN_MODULES=${BUILTIN_MODULES})
 set_target_properties(libproxy PROPERTIES PREFIX "" VERSION 1.0.0 SOVERSION 1)
+set_target_properties(libproxy PROPERTIES LINK_INTERFACE_LIBRARIES "")
 install(TARGETS libproxy DESTINATION ${LIB_INSTALL_DIR})
 install(FILES   proxy.h  DESTINATION ${INCLUDE_INSTALL_DIR})
-- 
1.7.2.3

